### PR TITLE
Module Enumerable ready for code review

### PR DESCRIPTION
**Pull request by tigamadou**

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
#### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi Amadou!  
### Amazing job!

Only one thing needs to be fixed. Please see my comment above


### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    count = 0
+    if !block_given?
+      count = length
+    elsif !number.nil?
+      my_each { |x| count += 1 if x == number }
+    else
+      my_each { |x| count += 1 if yield(x) }
+    end
+    count
+  end
```
Comment (Anna-Myzukina) [inline main.rb]: [REQUIRED]

Counts method works not correct.

        Failure/Error: expect(array.my_count(LOWEST_VALUE)).to eq 
          array.count(LOWEST_VALUE)
     
                expected: 11
                     got: 100

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Needs to fix one error in count method

**Comment by tigamadou** [comment]
count method edited.

**Review by Forison — APPROVED** [review]
## Hi @Amadou,
Good Job on implementing the change suggested by my colleague.
```my_count``` works fine💪 now
### Project is approved🙂
#### Happy coding🙂
#### @Forison

**Comment by tigamadou** [comment]
@Forison @Anna-Myzukina Thank you very much guys for your commitment 💪 .

**Pull request merged**